css/editorialBio.css

```diff
@@ -0,0 +1,12 @@
+.orcidContent{
+    display:flex;
+}
+
+.orcidIcon{
+    width:24px; 
```

Comment:
ORCID guidelines state:
* Scale the iD icon to match the line height of the surrounding text but go no smaller than 16×16.
* Allow for at least 50% buffer space around the icon. For the 16×16 pixel icon, this translates into a four-pixel margin on each side of the icon.

A better implementation could be:
```
  width: 1em;
  height: 1em;
  min-width: 16px;
  min-height: 16px;
  margin-left: .25em;
  margin-right: .25em;
```